.github/workflows/test.yml

```diff
@@ -23,6 +23,12 @@ jobs:
       with:
         php-version: ${{ matrix.php }}
 
+    - name: Cache Composer dependencies
+      uses: actions/cache@v2
+      with:
+        path: /tmp/composer-cache
+        key: ${{ runner.os }}-${{ hashFiles('**/composer.lock') }}
+
     - name: Install Dependencies
       uses: php-actions/composer@master
```

Comment:
Sorry have not had time to look at this. I wonder if it's because we're @master here for php-actions and master doesn't have 8 and 8.1 yet. Can you try `uses: php-actions/composer@v6` looks like from the docs https://github.com/php-actions/composer#using-different-versions-of-php-or-composer `@v6` and `@v7` both have 8.1 support. 

This is just a guess off the top of my head. This is the most i've been able to dig into this atm.

Reply:
@Olofguard I think the issues are the packages created by the workflow, you need to delete those and rerun the workflow. Here's the proof that all tests have passed. https://github.com/tenshiAMD/php-healthz/actions/runs/3169232332

Reply:
No actually looks the like issue is that on your branch it tries to push to our org which of course doesn't work as it's coming from your fork not from our repo. At least that's what I'm understanding atm. I'll investigate more when i have some more time.


Here's the error from the debug log 

```sh
Successfully tagged docker.pkg.github.com/generationtux/php-healthz/php-actions_composer_php-healthz:php-8.0-build2
  The push refers to repository [docker.pkg.github.com/generationtux/php-healthz/php-actions_composer_php-healthz]
  663422d6247b: Preparing
  692f464f5400: Preparing
  e3026b9684d6: Preparing
  0103c6155893: Preparing
  e8a1531411ad: Preparing
  b5ddaa9e9c6f: Preparing
  744437e91ddb: Preparing
  49a1a2a80ba6: Preparing
  cd3f5d78e589: Preparing
  994393dc58e7: Preparing
  b5ddaa9e9c6f: Waiting
  744437e91ddb: Waiting
  49a1a2a80ba6: Waiting
  cd3f5d78e589: Waiting
  994393dc58e7: Waiting
  denied: installation not allowed to Write organization package
```